src/app/components/mno-apps/mno-app.coffee

```diff
@@ -55,7 +55,7 @@ angular.module 'mnoEnterpriseAngular'
         vm.app ||= _.findWhere(apps, { id:  appId })
 
         # Init currency
-        vm.currency = MnoeOrganizations.selected.organization.billing_currency || MnoeConfig.marketplaceCurrency()
+        vm.currency = (if MnoeOrganizations.selected? then MnoeOrganizations.selected.organization.billing_currency else MnoeConfig.marketplaceCurrency())
```

Comment:
For this one I think it might be better to do something like this:
```
vm.currency = MnoeOrganizations.selected?.organization?.billing_currency || MnoeConfig.marketplaceCurrency()
```

This way, the currency will still fallback to `MnoeConfig.marketplaceCurrency` in the event that there is a selected organization but no billing_currency defined.

Reply:
@ouranos Actually, would it be necessary to guard against this since there should always be a billing currency set?

Reply:
Happy with what you suggest

Reply:
@adamaziz15 I've made the change as suggested. Cheers.